src/renderer/entities/network/model/network-model.ts

```diff
@@ -109,46 +185,60 @@ const createProviderFx = createEffect(({ chainId, nodes }: CreateProviderParams)
 
 type CreateApiParams = {
   chainId: ChainId;
-  provider: UniversalProvider;
+  provider: ProviderInterface;
 };
 const createApiFx = createEffect(async ({ chainId, provider }: CreateApiParams): Promise<ApiPromise | undefined> => {
-  try {
-    return networkService.createApi(provider);
-  } catch (e) {
-    console.log('error during create api', e);
+  if (provider.isConnected) {
+    try {
+      const api = await networkService.createApi(provider);
+
+      return api;
+    } catch (e) {
+      console.log('error during create api', e);
+    }
+  } else {
+    setTimeout(() => {
+      createApiFx({
+        chainId,
+        provider,
+      });
+    }, 1000);
   }
 });
 
-type ConnectParams = {
-  provider: UniversalProvider;
-  api: ApiPromise;
-};
-const connectFx = createEffect(async ({ provider, api }: ConnectParams): Promise<void> => {
-  await networkService.connect(provider, api);
-});
-
-type DisconnectParams = {
-  provider: UniversalProvider;
-  api: ApiPromise;
-  chainId: ChainId;
-};
-const disconnectFx = createEffect(async ({ provider, api, chainId }: DisconnectParams): Promise<ChainId> => {
-  await networkService.disconnect(provider, api);
-
-  return chainId;
-});
-
 forward({
-  from: kernelModel.events.appStarted,
+  from: combineEvents({
+    events: [kernelModel.events.appStarted, populateConnectionsFx.done],
```

Comment:
Are you sure about combined events?
Seems that `populateConnectionsFx.done` is enough because of chain call >>
```javascript
forward({
  from: kernelModel.events.appStarted,
  to: populateConnectionsFx,
});
```